pkgs/top-level/lua-generated-packages.nix

```diff
@@ -0,0 +1,279 @@
+/* ${GENERATED_NIXFILE} is an auto-generated file -- DO NOT EDIT! */
```

Comment:
Can you make you generator also to insert comment explaining how to regenerate this file in the header?